src/base/Cellar.sol

```diff
@@ -13,6 +13,8 @@ import { IGravity } from "../interfaces/IGravity.sol";
 import { AddressArray } from "src/utils/AddressArray.sol";
 import { Math } from "../utils/Math.sol";
 
+//TODO remove this
+import { Test, console } from "@forge-std/Test.sol";
```

Comment:
After you are done with testing, don't forget to clean up `console.log`s and delete this import.